doc/onboarding.md

```diff
@@ -0,0 +1,85 @@
+# Numberscope onboarding
+
+## A note on commands
+
+Unless otherwise specified, all commands we ask you to run are supposed to be
+run in a Terminal emulator using a shell like Bash or Zsh (on Linux and macOS)
+or in Git Bash (on Windows).
+
+## Accounts, etc.
+
+(Steps 2 and 3 only apply to the members of the CU Boulder Experimental
+Mathematics Lab.)
+
```

Comment:
Can we say "members of the Numberscope project at the CU Boulder Experimental Mathematics Lab"

Reply:
Addressed in https://github.com/numberscope/frontscope/pull/161/commits/e40351417c219ea94249a42405dec1079693be11.